account_move_name_sequence/models/account_move.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _compute_name(self):
+        for move in self.filtered(
+            lambda m: (m.name == "/" or not m.name)
+            and m.state == "posted"
+            and m.journal_id
+            and m.journal_id.sequence_id
```

Comment:
If you don't set the sequence_id field as required on the journal, it means that you could have mixed implementation. Is that ok? 

Personally, I would prefer having a consistent implementation across all journals.

Maybe it could be handled with a configuration.

Or maybe a simple note in the Readme:

```
Remarks:

- This module does not enforce the use of the Sequence for all journals. In case the Sequence field is not set on the Journal, then the module will fallback on the native behavior. If you want to enforce, you only need to override the `account.journal` and set the field sequence_id as required.
```


Actually, it is kind of required as you automatically create the sequence, but some funny user could remove the value from the field.

Reply:
In the code of v13, the sequence_id field has required=True in the field definition, cf https://github.com/odoo/odoo/blob/13.0/addons/account/models/account.py#L733
But, in the form view, it's required="0" cf https://github.com/odoo/odoo/blob/13.0/addons/account/views/account_view.xml#L274 in order to leave the field empty and let the inherit of create() auto-create the sequence.
When the company is created after the installation of the module, the sequence will be auto-created for every journal. But if you install the module after the creation of the company (which will happen for all existing v14 deployment that wish to go away of the sequence mess/bugs on journal entries), then you will have to create the sequence, unless if we provide a post-install script.
I'm OK to set the field as required and write a post-install script. But it means that you will have a rupture in the chronology of sequence right after the installation in all your journals and unless if you reconfigure all the journal sequence straight away after module installation. Like you, I also prefer to have a consistent implementation across all journals.

Reply:
@jcdrubay The sequence is now required and I added a post install script. I did that in a separate commit, so that we can easily remove it if other reviewers think it's a bad idea.